### ISSUE TITLE
Cannot handle CJK characters?

### ISSUE THREAD

**Issue by emisjerry** [federico-terzi/modulo#5]
File encoding is UTF-8:

```
layout: |
  Hello 中文測試 {{name}},
  This form is built with modulo!
```

Chinese characters cause problem, even Hello cannot display on the form.

**Comment by federico-terzi** [comment]
@emisjerry Thanks for reporting! I'll investigate

**Comment by federico-terzi** [comment]
Should be solved in next release thanks to #4 :)

**Comment by federico-terzi** [comment]
Released! Please update espanso to see the changes :)
